variants/dreamcatcher/platformio.ini

```diff
@@ -0,0 +1,27 @@
+[env:dreamcatcher] ; 2301, latest revision
+extends = esp32s3_base
+board = esp32s3box
```

Comment:
Just checking: is there a new board to add to /boards here?

Reply:
nope, there is a stock ESP32 solder core on the board, and this is it.